installer/README.md

```diff
@@ -0,0 +1,312 @@
+# Installer
+
+The best way to get started with Gitpod self-hosted
+
+> This guide assumes that a compatible infrastructure is in place. Please
+> refer to the [Gitpod docs](https://www.gitpod.io/docs/self-hosted/latest/installation)
+> for instructions on how to create the infrastructure.
+
+# Key Concepts
+
+The Installer is a complete replacement for our Helm charts. Over time,
+this had grown to be too complex to effectively support and was a barrier
+to entry for new users - the base config was many hundreds of lines long
+and did not have effective validation on it. By contrast, the Installer's
+config is < 50 lines long and can be fully validated before running.
+
+Conceptually, the process is now:
+1. generate a base config
+2. amend the config for your own use-case
+3. validate
+4. render the Kubernetes YAML
+5. `kubectl apply`
+
+# Quickstart
+
+## Create a gitpod-installer script
+
+```shell
+#!/bin/bash
+
+VERSION="${VERSION:-main.1844}" # Build ID taken from https://werft.gitpod-dev.com/
+
+docker run -it --rm \
+  -v $PWD:/gitpod \
+  -v $HOME/.kube:/root/.kube \
+  eu.gcr.io/gitpod-core-dev/build/installer:${VERSION} \
+  $@
+```
+
+Save this in your `$PATH` as `gitpod-installer` and make executable
```

Comment:
Where do folks normally put SH scripts for product? I feel like `$HOME/.gitpod/bin` isn't the right place..but there a better sub-folder to use than bin? I ask because I'm a long time Windows user now using Ubuntu. Can you think of any other products that persist scripts to user machines?

If we had it in bin (or whatever), we could do something like:

Add `gitpod-installer.sh` to your PATH with:

```bash
export PATH=$PATH:$HOME/.gitpod/bin
```

Reply:
This is one of those things that I think you could end up writing a whole readme on. As the plan is to change this to the [actual binary release](#6766) rather than a Bash script to wrap the Docker image, I sort of think we can keep this simple and assume an amount of knowledge from the reader.

Ultimately, my ambition is to be able to have something like how Helm does it with [binary](https://helm.sh/docs/intro/install/#from-the-binary-releases) and [script](https://helm.sh/docs/intro/install/#from-script)

Reply:
Nice, thank you for sharing. Actual binaries would rock, and, I like those helm examples - they seem like the right fit for where things are presently.